edx_exams/apps/api/test_utils/__init__.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework.test import APIClient, APITestCase
+
+from edx_exams.apps.api.test_utils.factories import UserFactory
+from edx_exams.apps.api.test_utils.mixins import JwtMixin
+from edx_exams.apps.core.models import ProctoringProvider
+
+TEST_USERNAME = 'api_worker'
+TEST_EMAIL = 'test@email.com'
+TEST_PASSWORD = 'QWERTY'
+
+
+class ExamsAPITestCase(JwtMixin, APITestCase):
+    """
+    Base class for API Tests
+    """
+
+    def setUp(self):
+        """
+        Perform operations common to all tests.
+        """
+        super().setUp()
+        self.create_user(username=TEST_USERNAME, email=TEST_EMAIL, password=TEST_PASSWORD, is_staff=True)
```

Comment:
is_staff is being changed to false somewhere, which is why I'm getting a 403 response.

Reply:
I think the problem is that, when you are converting the user information into JWT, you did not specify the user is staff or admin in the JWT token construction?

Reply:
Got it, so administrator should be `True`. Will update that.